cmd/uplink/cmd/cat.go

```diff
@@ -22,6 +22,10 @@ func init() {
 
 // catMain is the function executed when catCmd is called
 func catMain(cmd *cobra.Command, args []string) (err error) {
+	if *debugPprof {
```

Comment:
This should probably be added at a higher level, not inside each individual command.

Reply:
i was trying to figure that out, but couldn't think of a better place. where do you recommend?

Reply:
maybe `cmd/uplink/main.go`?

Reply:
yeah i could do that. egon?

Reply:
Yeah, `cmd/uplink/main.go` or you can set as `PreRunE` and `PostRunE` on `RootCmd`.

Reply:
PreRunE and PostRunE on RootCmd sounds cleaner

Reply:
hmm ok let me try one of those areas and see what looks best. thanks!

Reply:
I added the `PreRunE` and `PostRunE`.  I originally tried to add that on the `RootCmd` but they were **not** executed, I think because the `RootCmd` doesn't have a `RunE`, only the subcommands have a `RunE`.  Does that sound right?